Utilities/CMake/template-libHttpClient.Linux.C.vcxproj

```diff
@@ -1,115 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
PR description: "_Drop CMake project generation for Windows platforms as its no longer needed_"

Should this file remain for linux?

Reply:
I believe linux is not using a vcxproj for building at this point.  I will double check that.

Reply:
confirmed with Sebastian that this is not needed.  Has already been removed from main as well.